airbyte-webapp/src/components/base/DropDown/components/Option.tsx

```diff
@@ -45,10 +45,16 @@ export const OptionView = styled.div<{
 `;
 
 const Option: React.FC<IProps> = (props) => {
+  const dataTestId = props.data.testId
+    ? props.data.testId
+    : !["object", "array"].includes(typeof props.data.label)
+    ? props.data.label
+    : `select_value_${props.data.value}`;
```

Comment:
That's exactly the reason why I added this logic. By default, we were setting data-testid to label, so if we change it right now to value - it will break some e2e tests.

Should we open separate task for it? Or should I fix it as part of this PR?

Reply:
Let's push it into a separate PR.